src/tron/TronEngine.ts

```diff
@@ -367,6 +368,14 @@ export class TronEngine extends CurrencyEngine<TronTools, SafeTronWalletInfo> {
 
   async queryTransactionsInnerLoop(): Promise<void> {
     try {
+      // See if we need to add new data to the existing EdgeTransactions on disk
+      if (this.otherData.txListReset) {
+        // Clear out otherData.txQueryCache one time so we can re-process the
+        // transactions and populate the txInfo
+        this.otherData.txQueryCache.mainnet = asTxQueryCache({})
+        this.walletLocalDataDirty = true
+        this.otherData.txListReset = false
```

Comment:
Nitpicky (and feel free to ignore this) but it would read better to make all the changes to the otherData first and then mark it as dirty.